compose-builder/README.md

```diff
@@ -100,84 +108,110 @@ compose [options]
 Generates the EdgeX compose file as specified by options and stores them in the configured relese folder. Compose files are named appropriatly for release and options used to generate them.
 
 Options:
-    no-secty:   Generates non-secure compose file, otherwise generates secure compose file
-    arm64:      Generates compose file using ARM64 images
-    dev:        Generates compose file using local dev built images from edgex-go repo's 
-                'make docker' which creates docker images tagged with '0.0.0-dev'    
-    ds-bacnet:  Generates compose file with device-bacnet included
-    ds-camera:  Generates compose file with device-camera included
-    ds-grove:   Generates compose file with device-grove included (valid only with arm64 option)
-    ds-modbus:  Generates compose file with device-modbus included
-    ds-mqtt:    Generates compose file with device-mqtt included
-    ds-random:  Generates compose file with device-random included
-    ds-rest:    Generates compose file with device-rest included
-    ds-snmp:    Generates compose file with device-snmp included
-    ds-virtual: Generates compose file with device-virtual included
-    mqtt:       Generates compose file with services configure for MQTT Message Bus 
-    ui:         Generates stand-alone compose file for EdgeX UI	
+    no-secty:    Generates non-secure compose file, otherwise generates secure compose file
+    arm64:       Generates compose file using ARM64 images
+    dev:         Generates compose file using local dev built images from edgex-go repo's 
+                 'make docker' which creates docker images tagged with '0.0.0-dev'    
+    ds-bacnet:   Generates compose file with device-bacnet included
+    ds-camera:   Generates compose file with device-camera included
+    ds-grove:    Generates compose file with device-grove included (valid only with arm64 option)
+    ds-modbus:   Generates compose file with device-modbus included
+    ds-mqtt:     Generates compose file with device-mqtt included
+    ds-random:   Generates compose file with device-random included
+    ds-rest:     Generates compose file with device-rest included
+    ds-snmp:     Generates compose file with device-snmp included
+    ds-virtual:  Generates compose file with device-virtual included
+    modbus-sim:  Generates compose file with ModBus simulator included
+    asc-http:    Generates compose file with App Service HTTP Export included
+    asc-http-s:  Generates compose file with App Service HTTP Export Secrets included
```

Comment:
should this be there? remove it if not

Reply:
Opps, meant to remove. Good catch.

Reply:
Fixed & pushed.

Reply:
cool, that's where extra pair of eyes works :-)